src/main/java/uk/gov/pay/ledger/report/service/ReportService.java

```diff
@@ -29,19 +33,49 @@ public ReportService(ReportDao reportDao) {
         Map<String, Long> responseMap = new HashMap<>();
         Arrays.stream(TransactionState.values()).forEach(state -> responseMap.put(state.getStatus() , 0L));
 
-        reportDao.getPaymentCountsByState(params).forEach(result ->
+        TransactionStatisticQuery transactionStatisticQuery = buildBaseTransactionStatisticQuery(params);
+
+        reportDao.getPaymentCountsByState(transactionStatisticQuery).forEach(result ->
                 responseMap.put(TransactionState.from(result.getState()).getStatus(), result.getCount()));
 
         return responseMap;
     }
 
     public TransactionSummaryResult getTransactionsSummary(TransactionSummaryParams params) {
-        TransactionsStatisticsResult payments = reportDao.getTransactionSummaryStatistics(params, TransactionType.PAYMENT);
-        TransactionsStatisticsResult refunds = reportDao.getTransactionSummaryStatistics(params, TransactionType.REFUND);
-        return new TransactionSummaryResult(payments, refunds, payments.getGrossAmount() - refunds.getGrossAmount());
+        TransactionStatisticQuery transactionStatisticQuery = buildBaseTransactionStatisticQuery(params);
+
+        TransactionsStatisticsResult payments = reportDao.getTransactionSummaryStatistics(transactionStatisticQuery, TransactionType.PAYMENT);
+        TransactionsStatisticsResult refunds = reportDao.getTransactionSummaryStatistics(transactionStatisticQuery, TransactionType.REFUND);
+        TransactionsStatisticsResult motoPayments = new TransactionsStatisticsResult(0L, 0L);
+
+        if (params.isIncludeMotoStatistics()) {
+            TransactionStatisticQuery motoQuery = transactionStatisticQuery.withMoto(params.isIncludeMotoStatistics());
```

Comment:
I think that this should either be `transactionStatisticQuery.withMoto(true)` or possibly `withMoto` shouldn't take parameters. It's exactly the same in practice, but maybe slightly confusing to read as it is.